deploy/roles/common/tasks/main.yml

```diff
@@ -80,3 +80,27 @@
   group:
     name: "{{ restricted_group_name }}"
     state: present
+    
+- name: Create empty log files if they don't exist
+  file:
+    path: "{{ item }}"
+    state: touch
+    owner: root
+    group: root
+    mode: 0640
+  with_items: "{{ log_file_locations }}"
```

Comment:
Hey @tanderegg are you sure this will always exist? See travis failure - https://travis-ci.org/stirlingv/aurora/jobs/229248009